src/warc2zim/converter.py

```diff
@@ -109,7 +115,20 @@ def __init__(self, args):
         self.long_description = args.long_description
         self.creator_metadata = args.creator
         self.publisher = args.publisher
-        self.tags = DEFAULT_TAGS + (args.tags or [])
+        self.tags = [
+            tag
+            for tag in list(
+                set(
+                    DEFAULT_TAGS
+                    + [
+                        t2.strip()
+                        for t1 in args.tags.split(",")  # support comma
```

Comment:
We already agreed on not supporting multiple formats. Serves no purpose and brings confusion.
We also agreed to use semi colons, as it's what is stored in Metadata so we only ever support semicolon separated values.

Reply:
That's a great decision, I unfortunately found at least two scrapers which where support only the comma and I felt it was awkward since Metadata support only semicolon. Should be shared at zimscraperlib level in the end.